src/discounts/filters/useFilterLeftOperandsProvider.ts

```diff
@@ -0,0 +1,29 @@
+import { LeftOperand } from "@dashboard/components/ConditionalFilter/LeftOperandsProvider";
+import { AttributeInputTypeEnum } from "@dashboard/graphql";
+import unionBy from "lodash/unionBy";
+import { useState } from "react";
+
+export const STATIC_OPTIONS: LeftOperand[] = [
+  {
+    value: "startDate",
+    label: "Start date",
+    type: AttributeInputTypeEnum.DATE_TIME,
+    slug: "startDate",
+  },
+  {
+    value: "endDate",
+    label: "End date",
+    type: AttributeInputTypeEnum.DATE_TIME,
+    slug: "endDate",
+  },
+];
+
+export const useFilterLeftOperandsProvider = () => {
```

Comment:
this is copy-pasted file, why can't you re-use it? If the only difference are the STATIC_OPTIONS, just pass it as argument to `useFilterLeftOperandsProvider`